frontend/src/components/OurProjects.vue

```diff
@@ -1,72 +1,91 @@
 <template>
   <v-container fluid class="white-container">
-    <v-layout row wrap>
-      <v-flex
-        id="items"
-        v-for="project in projects"
-        :key="project.name"
-        xs12
-        md6
-        lg4
-        xl3
+    <v-row>
+      <v-col>
+        <v-card flat color="transparent">
+          <v-card-text class="title">
+            <p class="blue-title">NASZE PROJEKTY</p>
+          </v-card-text>
+        </v-card>
+      </v-col>
+    </v-row>
+    <v-row wrap>
+      <v-col
+        class="items"
+        v-for="(project, index) in projects"
+        :key="index"
+        cols="12"
+        xs="12"
+        md="6"
+        lg="4"
+        xl="3"
       >
         <v-item-group>
-          <v-hover>
-            <v-card slot-scope="{ hover }">
-              <v-img :src="project.image" aspect-ratio="1.9"></v-img>
+          <v-hover v-slot="{ hover }">
+            <v-card @click.stop="clickImage(project)">
+              <v-img :src="project.imageAdress" aspect-ratio="1.9" />
+              <v-card-title class="card">
+                {{ project.name }}
+              </v-card-title>
               <v-expand-transition>
-                <div v-if="hover" id="card--reveal">
-                  <v-img id="hoverd-img" :src="hoveredImg"></v-img>
+                <div v-if="hover" class="card--reveal">
+                  <v-img class="card--hover" :src="hoveredImg" />
                 </div>
               </v-expand-transition>
             </v-card>
           </v-hover>
         </v-item-group>
-      </v-flex>
-    </v-layout>
+      </v-col>
+    </v-row>
+    <v-dialog v-model="dialog" width="50rem">
```

Comment:
that's huge width. Maybe percentage would be better?

Reply:
It was there because of the large screen view when the dialog was getting extended throughout the screen making it look ridiculous. I changed it to max-width="50rem" to make it more elegant.